frontend/fingertips-frontend/components/organisms/PopulationPyramidChartTable/PopulationDataTable.tsx

```diff
@@ -36,8 +28,9 @@ const valueFormatter = (value: number | string | undefined) => {
 const StyledAreaNameHeader = styled(StyledAlignLeftHeader)(
   typography.font({ size: 19 }),
   {
-    borderTop: `solid #F3F2F1 2px`,
+    borderTop: 'solid #F3F2F1 2px',
```

Comment:
```suggestion
    borderTop: `solid ${GovukColours.LightGrey} 2px`,
```
although we have hard coded this colour in borders all over the project